.golangci.yml

```diff
@@ -10,7 +13,15 @@ linters-settings:
       modules:
         - github.com/pkg/errors:
             recommendations:
-            - errors
+              - errors
+  forbidigo:
+    forbid:
+      - Fatal(f|ln)?$
```

Comment:
This needs to be removed for DTLS. The protocol itself defines a `Fatal` alert level that must be sent to the other party in case certain errors occur.

Reply:
I see.. Let me check what I can do. Would you be fine with using `//nolint` directives?

Otherwise, I need to build a workaround which would allow us to customize the `golangci-lint.yaml` per-repo..

Reply:
If the `//nolint` ones work that's totally fine

Reply:
Argh, there are 116 occurences of `alert.Fatal`. Adding exceptions for all of them is probably not the best way here :(

Reply:
I changed the regex which catches `Fatal` usage. It should work now.

Reply:
Nice!